zerver/tests/test_notification_data.py

```diff
@@ -212,6 +212,24 @@ def test_is_notifiable(self) -> None:
         user_data = self.create_user_notifications_data_object(user_id=user_id, pm_push_notify=True)
         self.assertTrue(user_data.is_notifiable(acting_user_id=acting_user_id, idle=True))
 
+    def test_bot_user_notifiability(self) -> None:
+        # Non-bot user (`user_id=9`) should get notified, while bot user (`user_id=10`) shouldn't
+        for user_id, notifiable in [(9, True), (10, False)]:
+            user_data = UserMessageNotificationsData.from_user_id_sets(
+                user_id=user_id,
+                flags=["mentioned"],
+                private_message=True,
+                online_push_user_ids=set(),
+                pm_mention_email_disabled_user_ids=set(),
+                pm_mention_push_disabled_user_ids=set(),
+                all_bot_user_ids={10, 11, 12},
```

Comment:
This list of IDs also seems fragile.  Can we pull the same trick of hard-coding email addresses, and convert those to IDs for this set?